modal-go/task_command_router_client.go

```diff
@@ -0,0 +1,629 @@
+package modal
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log/slog"
+	"net"
+	"net/url"
+	"strings"
+	"sync/atomic"
+	"time"
+
+	pb "github.com/modal-labs/libmodal/modal-go/proto/modal_proto"
+	"golang.org/x/sync/singleflight"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
+
+// tlsCredsNoALPN is a TLS credential that skips ALPN enforcement.
```

Comment:
Can we include a comment about what interface that `tlsCredsNoALPN` is implementing? I suspect it's https://pkg.go.dev/google.golang.org/grpc@v1.78.0/credentials#TransportCredentials